spec/views/form_drafts/show_spec.rb

```diff
@@ -0,0 +1,165 @@
+require 'rails_helper'
+include RSpecHtmlMatchers
+
+describe 'form_drafts/show.haml' do
+  before :each do
+    @draft = create :form_draft
+  end
+  it 'contains the name of the form draft' do
+    render
+    expect(rendered).to have_tag 'h1', with: { class: 'title' } do
+      with_text(/#{@draft.name}/)
+    end
+  end
+  it 'gives headings a class of heading' do
+    heading_field = create :field, form_draft: @draft, data_type: 'heading'
```

Comment:
I'd like to see some sort of comment at the top of the file explaining how in each of these tests, the field you create within the test is the only field belonging to the form draft, and that's why the tests are a little non-specific in how they find things.